src/components/Outcome/OutcomeCategorical/TrendingOutcomeCategorical/index.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import cn from 'classnames'
+import style from '../outcomeCategorical.scss'
+
+const cx = cn.bind(style)
+
+const TrendingOutcomeCategorical = ({
+  entryStyle, outcome, percentage, resolutionDate,
+}) => (
+  <div className="outcomes outcomes--categorical">
```

Comment:
`cx('outcomes', 'categorical')`

if we're going for css-modules, we need to use the bound styles everywhere

Reply:
👌